tests/e2e-cypress/support/commands.js

```diff
@@ -83,22 +83,55 @@ const addWhat3WordFixtureAndIntercept = () => {
     }).as('coordinates-for-w3w')
 }
 
-// Adds a command that visit the main view and wait for the map to be shown (for the app to be ready)
+export function getDefaultFixturesAndIntercepts() {
+    return {
+        addLayerTileFixture,
+        addFileAPIFixtureAndIntercept,
+        addLayerFixtureAndIntercept,
+        addTopicFixtureAndIntercept,
+        addCatalogFixtureAndIntercept,
+        addHeightFixtureAndIntercept,
+        addWhat3WordFixtureAndIntercept,
+    }
+}
+
+/**
+ * Command that visits the main view and waits for the map to be shown (for the app to be ready) All
+ * parameters are optional. They can either be passed in order or inside a wrapper object.
+ *
+ * @param {String} lang Language of the app
+ * @param {Object} otherParams URL parameters except the language
+ * @param {Boolean} withHash Wheather or not to use the new URL format (that has a hash)
+ * @param {Object} geolocationMockupOptions Latitude and Longitude of facked geolocation
+ * @param {Object} fixturesAndIntercepts Contains functions that add interceptions to the backend
+ */
 Cypress.Commands.add(
     'goToMapView',
     (
         lang = 'en',
         otherParams = {},
         withHash = false,
-        geolocationMockupOptions = { latitude: 47, longitude: 7 }
+        geolocationMockupOptions = { latitude: 47, longitude: 7 },
+        fixturesAndIntercepts = getDefaultFixturesAndIntercepts()
     ) => {
-        addLayerTileFixture()
-        addFileAPIFixtureAndIntercept()
-        addLayerFixtureAndIntercept()
-        addTopicFixtureAndIntercept()
-        addCatalogFixtureAndIntercept()
-        addWhat3WordFixtureAndIntercept()
-        addHeightFixtureAndIntercept()
+        // If parameters have been passed inside a wrapper object, unwrap the object
+        if (typeof lang === 'object' && !(lang instanceof String)) {
+            const params = lang
+            return cy.goToMapView(
+                params.lang,
+                params.otherParams,
+                params.withHash,
+                params.geolocationMockupOptions,
+                params.fixturesAndIntercepts
+            )
+        }
+        fixturesAndIntercepts.addLayerTileFixture()
+        fixturesAndIntercepts.addFileAPIFixtureAndIntercept()
+        fixturesAndIntercepts.addLayerFixtureAndIntercept()
+        fixturesAndIntercepts.addTopicFixtureAndIntercept()
+        fixturesAndIntercepts.addCatalogFixtureAndIntercept()
+        fixturesAndIntercepts.addWhat3WordFixtureAndIntercept()
+        fixturesAndIntercepts.addHeightFixtureAndIntercept()
```

Comment:
If you want I can do this, although I think what you describe will never happen, as the tests are not supposed to make calls to the backend. But if you pass an array with missing intercepts, this would happen.

Reply:
Not necessarily that, but at least give a comprehensive feedback to anyone giving an object with missing intercepts, if you forgot to declare some I don't think the error will be very clear